msal/src/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -51,7 +51,7 @@ public SilentRequest(AuthenticationRequestParameters authenticationRequestParame
         protected override void SetAdditionalRequestParameters(OAuth2Client client)
         {
             client.AddBodyParameter(OAuth2Parameter.GrantType, OAuth2GrantType.RefreshToken);
-            client.AddBodyParameter(OAuth2Parameter.RefreshToken, _msalRefreshTokenItem.RefreshToken);
+            client.AddBodyParameter(OAuth2Parameter.RefreshToken, _msalRefreshTokenItem.Secret);
```

Comment:
>Secret [](start = 88, length = 6)

why this name?

Reply:
According to new schema AT, RT and IdTokens are called credentials and all of them have Secret - which is basically content of token

Reply:
so just to be more consistent with new schema

Reply:
thanks

---
In reply to: [191035446](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1065#discussion_r191035446) [](ancestors = 191035446)